src/OrleansSQLUtils/CreateOrleansTables_MySql.sql

```diff
@@ -185,19 +184,13 @@ CREATE TABLE OrleansQuery
 		' 
 		INSERT INTO OrleansMembershipVersionTable
 		(
-			DeploymentId,
-			Version
+			DeploymentId
 		)
-		SELECT	
-			@deploymentId,
-			@version
-		WHERE NOT EXISTS
-		(			
-			SELECT 1
-			FROM OrleansMembershipVersionTable
-			WHERE DeploymentId = @deploymentId AND @deploymentId IS NOT NULL
-			FOR UPDATE
-		);
+		Values
+		(
+			@deploymentId
+		)
+		ON DUPLICATE KEY UPDATE DeploymentId = DeploymentId;
```

Comment:
I've made sure to follow the expected contract:
* new row inserted  ==> return 1.
* no row was changed ==> return 0.
* 2 will never happen.